UiPath.PowerShell/Cmdlets/GetRobot.cs

```diff
@@ -35,9 +36,22 @@ public class GetRobot:FilteredIdCmdlet
         [Parameter(ParameterSetName = "Filter")]
         public string HostingType { get; set; }
 
+        [ValidateEnum(typeof(Web.Client20184.Models.RobotDtoCredentialType))]
+        [Filter]
+        [Parameter(ParameterSetName = "Filter")]
+        public string CredentialType { get; set; }
```

Comment:
Unfortunately Orchestrator is not supporting odata queries on CredentialType 👎 . We should remove the parameter from this cmdlet